web/styles/left_sidebar.css

```diff
@@ -338,6 +338,12 @@ ul.filters {
     li.out_of_home_view {
         color: hsl(0deg 0% 20% / 50%);
 
+        &:has(li.unmuted_topic .unread_mention_info) {
```

Comment:
Has selectors are complex, possibly expensive and probably best to avoid -- we control what CSS classes we're adding for this situation, so we should be able to do simpler CSS here.

Also https://developer.mozilla.org/en-US/docs/Web/CSS/:has suggests Firefox doesn't support it?

Reply:
Updated PR to remove has selector